2.ServiceTracker_App/ComponentSchematic/tracker-data-component.yaml

```diff
@@ -8,38 +8,80 @@ spec:
   arch: amd64
   parameters:
```

Comment:
The `workloadType` for this component will need to be updated to be a `ContainerizedWorkload`.

Reply:
Will do in separate PR.